DOC-5730 Update RBAC links and terminology (#173)
datastax/astra-streaming-docs: modules/ROOT/pages/astream-org-permissions.adoc

:navtitle: Manage roles and permissions

You manage role-based access control (RBAC) for {product} through your {astra} organization.
For more information about {astra} RBAC, see xref:astra-db-serverless:administration:rbac.adoc[].

== {astra} roles and permissions for {product}

To access {product}, the minimum required permissions are as follows:

* **{manage-streaming-permission} (`org-stream-manage`)**: Create, read, update, and delete {product} resources in the {astra-ui} and with the APIs.
* **{view-db-permission} (`org-db-view`)**: View the {astra-ui}.
This permission is required for a user to access {product} through the {astra-ui}.
This permission isn't required for programmatic access.

> By default, {astra} has no roles that are scoped exclusively to {product}.

Reviewer: ```suggestion
By default, {astra} has no roles specific to {product}.
```

Reviewer: Optional - maybe more direct and concise
